resources/qml/ImageButton.qml

```diff
@@ -11,7 +12,7 @@ AbstractButton {
 		id: buttonImg
 		// Workaround, can't get icon.source working for now...
 		anchors.fill: parent
-		source: "image://colorimage/" + image + "?" + (button.hovered ? colors.highlight : colors.buttonText)
+		source: src ? src : ("image://colorimage/" + image + "?" + (button.hovered ? colors.highlight : colors.buttonText))
```

Comment:
Using 'image://..' resulted in a greyed out icon appearance.

Reply:
That's probably because the colors.buttonText is grey. I think the proper solution in that case is not to allow overriding the src, but to make the highlight and normal colour a property and allow overriding them. I think that would be useful in a few other places as well in the future.

Reply:
Thanks. Made that change.